index.js

```diff
@@ -190,9 +192,13 @@ function choo () {
 // initial application state model
 // obj -> obj
 function appInit (opts) {
+  const initialLocation = (opts.hash === true)
+    ? hashMatch(document.location.hash)
+    : document.location.href
+
   const model = {
     namespace: 'app',
-    state: { location: document.location.href },
+    state: { location: initialLocation },
     subscriptions: [],
     reducers: {
       // handle href links
```

Comment:
Oh yeah, that was to support `<a href="#"></a>` style links; not sure it's needed anymore, but eh - let's not remove it quite yet :sparkles: